addons/binding/org.openhab.binding.zoneminder/ESH-INF/thing/server-bridge.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="zoneminder"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 org.eclipse.smarthome.thing-description.xsd">
+
+    <bridge-type id="server">
+        <label>ZoneMinder Server</label>
+        <description>ZoneMinder Server</description>
+        <channels>
+            <channel id="is-alive" typeId="server_is_alive">
+                <label>Is Alive</label>
+                <description>ON / OFF value to determine from OpenHAB if the server is actually up and running.</description>
+            </channel>
+
+            <channel id="cpu-load" typeId="server_cpu_load">
+                <label>CPU load</label>
+                <description>CPU load on ZoneMinder server</description>
+            </channel>
+            
+            <channel id="disk-usage" typeId="server_disk_usage">
+                <label>Disk Usage</label>
+                <description>Disk USage on ZoneMinder server</description>
+            </channel>
+
+            <channel id="zm-version" typeId="server_zm_version">
+                <label>Server Version</label>
+                <description>Version number of the ZoneMinder Server</description>
+            </channel>
+            
+            <channel id="zm-api-version" typeId="server_zm_api_version">
```

Comment:
this is a static information, which should be put into a Thing property by the handler. The channel should be removed.